prework.md

```diff
@@ -0,0 +1,17 @@
+## My questions from the reading
+### Heap
+* "In order to maintain the heap order property, all we need to do is swap the root with its smallest child less than the root." why the smallest child less than the root? what happens if we swap it with the larger child?
```

Comment:
Let's say the smaller child is 2 and the larger child is 4.  If you swap the root with the larger child, now you have 4 as the parent of 2, which breaks the heap order property!